tests/RuleSet/RuleSetTest.php

```diff
@@ -336,57 +338,53 @@ public function testGetMissingRuleConfiguration(): void
         $ruleSet->getRuleConfiguration('_not_exists');
     }
 
-    public function testDuplicateRuleConfigurationInSetDefinitions(): void
+    /**
+     * @dataProvider provideAllSetCases
+     */
+    public function testDuplicateRuleConfigurationInSetDefinitions(RuleSetDescriptionInterface $set): void
```

Comment:
```php
// @PER
['method_argument_space' => X];

// @Symfony
['method_argument_space' => Y];

// @PhpCsFixer
[
    '@PER' => true,
    '@Symfony' => true,
    'method_argument_space' => X,
];
```

With the above rule sets, this test would fail because of `method_argument_space` configuration in `@PhpCsFixer` being the same as in parent rule set (`@PER`), which was incorrect, because it would be overridden by Symfony.